docs/user-guide/configuring/language-options.md

```diff
@@ -0,0 +1,214 @@
+            "node": true
+        }
+    }
+}
+```
+
+And in YAML:
+
+```yaml
+---
+  env:
+    browser: true
+    node: true
+```
+
+### Using a plugin
+
+If you want to use an environment from a plugin, be sure to specify the plugin name in the `plugins` array and then use the unprefixed plugin name, followed by a slash, followed by the environment name. For example:
+
+```json
+{
+    "plugins": ["example"],
+    "env": {
+        "example/custom": true
+    }
+}
+```
+
+Or in a `package.json` file
+
+```json
+{
+    "name": "mypackage",
+    "version": "0.0.1",
+    "eslintConfig": {
+        "plugins": ["example"],
+        "env": {
+            "example/custom": true
+        }
+    }
+}
+```
+
+## Specifying Globals
+
+The [no-undef](https://eslint.org/docs/rules/no-undef) rule will warn on variables that are accessed but not defined within the same file. If you are using global variables inside of a file then it's worthwhile to define those globals so that ESLint will not warn about their usage. You can define global variables either using comments inside of a file or in the configuration file.
+
+### Using configuration comments
+
+To specify globals using a comment inside of your JavaScript file, use the following format:
+
+```js
+/* global var1, var2 */
+```
+
+This defines two global variables, `var1` and `var2`. If you want to optionally specify that these global variables can be written to (rather than only being read), then you can set each with a `"writable"` flag:
+
+```js
+/* global var1:writable, var2:writable */
+```
+
+### Using configuration files
+
+To configure global variables inside of a configuration file, set the `globals` configuration property to an object containing keys named for each of the global variables you want to use. For each global variable key, set the corresponding value equal to `"writable"` to allow the variable to be overwritten or `"readonly"` to disallow overwriting. For example:
+
+```json
+{
+    "globals": {
+        "var1": "writable",
+        "var2": "readonly"
+    }
+}
+```
+
+And in YAML:
+
+```yaml
+---
+  globals:
+    var1: writable
+    var2: readonly
+```
+
+These examples allow `var1` to be overwritten in your code, but disallow it for `var2`.
+
+Globals can be disabled with the string `"off"`. For example, in an environment where most ES2015 globals are available but `Promise` is unavailable, you might use this config:
+
+```json
+{
+    "env": {
+        "es6": true
+    },
+    "globals": {
+        "Promise": "off"
+    }
+}
+```
+
+For historical reasons, the boolean value `false` and the string value `"readable"` are equivalent to `"readonly"`. Similarly, the boolean value `true` and the string value `"writeable"` are equivalent to `"writable"`. However, the use of older values is deprecated.
+
+**Note:** Enable the [no-global-assign](https://eslint.org/docs/rules/no-global-assign) rule to disallow modifications to read-only global variables in your code.
```

Comment:
Again, not sure we want to reference specific rules here in case things change. We can always reference from the rule docs to this page.

Reply:
Please let me know if one of the following changes works in this case:

Suggestion one: deleting the note.

Suggestion two:

> **Note:** You can enable some ESLint [rules] (https://eslint.org/docs/rules) to disallow modifications to read-only global variables in your code.

Reply:
Friendly ping @nzakas

Reply:
```suggestion
```

Reply:
I think it makes sense to remove it.